tests/e2e/TestConstants.ts

```diff
@@ -238,5 +238,13 @@ export const TestConstants = {
     /**
      * Running test suite - possible variants can be found in package.json scripts part.
      */
-    TEST_SUITE: process.env.TEST_SUITE || 'test-happy-path'
+    TEST_SUITE: process.env.TEST_SUITE || 'test-happy-path',
+
+    /**
+     * The repo (with README.md in root) and access token are needed for to run test-git-ssh
+     */
+    TS_GITHUB_TEST_REPO: process.env.TS_GITHUB_TEST_REPO || '',
+
+    TS_GITHUB_TEST_REPO_ACCESS_TOKEN: process.env.TS_GITHUB_TEST_REPO_ACCESS_TOKEN || ''
```

Comment:
It would be great to add a comment with explanation, like for the previous env variable

Reply:
Comment was added:
https://github.com/eclipse/che/pull/15561/commits/969bc66d11d9faf15206931d35d673796171a52f#diff-a8a79627cfd921d870ae2b4dfdf68b57R244